cli/main.go

```diff
@@ -49,5 +55,7 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	nextCommands()
+	if len(os.Args) == 1 {
```

Comment:
Wasn't sure what this was for; add inline documentation above this block, informing the reader that calling `./cli` launches interactive mode.